robots:
  a_khi:
    default_gripper: a_khi_gripper
  b_khi:
    default_gripper: b_khi_gripper

grippers:
  a_khi_gripper:
    type:          RobotiqGripper
    controller_ns: a_khi_gripper_controller
    max_effort:    5.0
    velocity:      0.1
  b_khi_gripper:
    type: SuctionGripper
    controller_ns: /b_khi_gripper_controller
